Sources/MapboxNavigation/NavigationMapView+Annotations.swift

```diff
@@ -35,15 +35,17 @@ extension NavigationMapView {
         guard style.image(withId: "RouteInfoAnnotationLeftHanded") == nil,
               style.image(withId: "RouteInfoAnnotationRightHanded") == nil else { return }
         
+        let bottomInset = UIApplication.shared.keyWindow?.safeAreaInsets.bottom ?? 0
+        
         // Right-hand pin
         if let image = Bundle.mapboxNavigation.image(named: "RouteInfoAnnotationRightHanded") {
             // define the "stretchable" areas in the image that will be fitted to the text label
             // These numbers are the pixel offsets into the PDF image asset
-            let stretchX = [ImageStretches(first: Float(33), second: Float(52))]
-            let stretchY = [ImageStretches(first: Float(32), second: Float(35))]
+            var stretchX = [ImageStretches(first: Float(33), second: Float(52))]
```

Comment:
The `stretchX`, `stretchY` and `imageContent` don't change their values, we don't need to update them as variables.